src/backend/joanie/core/admin.py

```diff
@@ -120,12 +120,14 @@ class ContractDefinitionAdmin(admin.ModelAdmin):
 class ContractAdmin(admin.ModelAdmin):
```

Comment:
We should add `address` into `OrganizationAdmin`.

Reply:
You mean in the list_display ?

Reply:
Ah nope I believed we customized fields displayed in change view.